run-tests.sh

```diff
@@ -31,6 +31,7 @@ trap cleanup EXIT
 
 python -m check_manifest --ignore ".*-requirements.txt"
 python -m sphinx.cmd.build -qnNW docs docs/_build/html
+python -m pytest
```

Comment:
I would add a comment here to explain that, without the path to the tests folder, it is simply checking style in docs and code folders